fli-iam/shanoir-ng
# shanoir-issue#1302 Subject removal from study - orphan subject

(Sorry @jcomedouteau I messed up with your pr trying to merge develop si reverted it here)

How to test:

Add two subject (A & B) on a study 1
Add subject B to study 2
Go to study 1 detail, edit it and remove subject A and B.
Subject A should have been deleted from Shanoir along with its data
Subject B should still be present on study 2 but not study 1
Closes https://github.com/fli-iam/shanoir-ng/issues/1302
